gen/gen.go

```diff
@@ -119,23 +146,69 @@ func Gen(exercise string, j interface{}, t *template.Template) error {
 	return ioutil.WriteFile(filepath.Join(dirExercise, "cases_test.go"), src, 0666)
 }
 
-func getPath(jFile string) (jPath, jOrigin, jCommit string) {
+func getLocal(jFile string) (jPath, jOrigin, jCommit string) {
 	// Ideally draw from a .json which is pulled from the official x-common
 	// repository.  For development however, accept a file in current directory
 	// if there is no .json in source control.  Also allow an override in any
 	// case by environment variable.
-	if jPath = os.Getenv("EXTEST"); jPath > "" {
+	if jPath := os.Getenv("EXTEST"); jPath > "" {
```

Comment:
I hadn't considered this, good point.
As it is now, if this happened you would get output like so:
```
> EXTEST="doesnt_exist" go run exercises/leap/.meta/gen.go 
2017/04/15 07:00:36 [LOCAL] fetching leap test data
2017/04/15 07:00:36 [LOCAL] source: doesnt_exist
2017/04/15 07:00:36 [LOCAL] No test data found
2017/04/15 07:00:36 [REMOTE] fetching leap test data
2017/04/15 07:00:37 [REMOTE] source: github.com/exercism/x-common
```
which let's you know that there is no test data in the file so you can then check it and fix the env if necessary. Though maybe a quick check in the `getLocal` function, that the file exists would be helpful. I'll look into it, thanks @tleen